aldryn_faq/models.py

```diff
@@ -68,7 +69,9 @@ class Meta:
 
     def __str__(self):
         if six.PY2:
-            return self.safe_translation_getter('name', default=unicode(self.pk))
+            return self.safe_translation_getter(
+                'name', default=six.u(str(self.pk))
+            )
```

Comment:
This could be DRYer by using a var to hold the default before you enter the if/else block